sklearn/gaussian_process/gaussian_process.py

```diff
@@ -566,20 +565,14 @@ def reduced_likelihood_function(self, theta=None):
             A dictionary containing the requested Gaussian Process model
             parameters:
 
-                sigma2
```

Comment:
I'll check a bit more but the bottom line is that it leads to a  ["too deeply nested" error](http://www.tex.ac.uk/FAQ-toodeep.html).